### PR TITLE
Add a warning about ZHA and Sonoff ZBBridge

### PR DESCRIPTION
**Pull request by Adminiuga**
## Proposed change

Using Sonoff ZBBridge with ZHA relies on using serial connection over the wifi network. Sonoff ZBBridge uses EZSP protocol and if there's a drop in communication because of the wireless connection, than the protocol has to be restarted. Warn users about it, in a desperate attempt to stop them from opening issues about it. 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.
git
## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/zha.markdown [hunk 1]
```diff
@@ -75,6 +75,11 @@ Some other Zigbee coordinator hardware may not support a firmware that is capabl
   - [PiZiGate](https://zigate.fr/produit/pizigate-v1-0/)
   - [Wifi ZiGate](https://zigate.fr/produit/zigate-pack-wifi-v1-3/)
 
+<div class="note warning">
+<b>EZSP</b> protocol requires a stable connection to the serial port. With <em>ITEAD Sonoff ZBBridge</em> connecting over the WiFi network
+it is expected to see <code>NCP entered failed state. Requesting APP controller restart</code> in the logs. This is a normal part of operation and indicates there was a drop in communication between ZHA and SonOff bridge.
+</div>
```
Comment (frenck) [inline source/_integrations/zha.markdown]:
```suggestion
<div class="note warning">

The **EZSP** protocol requires a stable connection to the serial port. With _ITEAD Sonoff ZBBridge_ connecting over the WiFi network
it is expected to see `NCP entered failed state. Requesting APP controller restart` in the logs. This is a normal part of the operation and indicates there was a drop in communication between ZHA and SonOff bridge.

</div>
```

**Review by frenck — CHANGES_REQUESTED** [review]
Small suggested tweak, to minimize HTML usage as much as possible.

**Comment by Adminiuga** [comment]
yeah, my bad. For some reason pre-view was slow to pick the changes, which made me think it did not like MD inside the tags.

**Review by frenck — APPROVED** [review]
Thanks! 👍 

It can switch HTML/Markdown context, important is to leave an empty line between switching such a context. (hence the new line before and after the div)

**Pull request merged**